dockge/umbrel-app.yml

```
@@ -0,0 +1,64 @@
+manifestVersion: 1
+id: dockge
+category: developer
+name: Dockge
+version: "1.4.2"
+tagline: Run custom Docker containers on your Umbrel
+description: >-
+  ⚠️ Make sure to only use named Docker volumes for your stacks and containers. Data in bind-mounted volumes
+  will be lost when the Portainer app is restarted or updated.
+
+
+  ⚠️ Watch out for port conflicts between your custom Docker containers and your umbrelOS apps.
+
+
+  Dockge is a fancy, easy-to-use and reactive self-hosted docker compose.yaml stack-oriented manager to run custom Docker 
+  containers on your Umbrel.
+
+  🛠️ Dockge on Umbrel is for power users, follow these best practices to avoid issues:
+
+
+  1. Data persistence: Make sure to only used named Docker volumes for your stacks and containers. Data in bind-mounted
+  volumes will be lost when the Portainer app is restarted or updated.
+
+
+  2. Port management: Watch out for potential port conflicts between your custom containers and umbrelOS' service containers,
+  apps you have installed from the Umbrel App Store or Community App Stores, and any apps you go to install in the future.
+
+
+  3. Container restart policy: Set your containers to "unless-stopped" or "always" restart policies. This will allow your containers
+  to restart automatically when the Portainer app is restarted or updated.
+
+
+  4. Web access to containers: Access your custom containers in your browser at umbrel.local:<mapped-port>. For example, for a container
+  with a web UI running on port 4545, navigate to umbrel.local:4545 to access it.
+website: https://dockge.kuma.pet
+dependencies: []
+repo: https://github.com/louislam/dockge
+support: https://github.com/louislam/dockge/issues
+port: 5001
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
+releaseNotes: >-
+  🐛 Bug Fixes
+
+    #380 Fixed the startup issue that Dockge is no longer working after restart
+    #342 Fixed the login placeholder issue (Thanks @cyril59310)
+
+  🦎 Translation Contributions
+
+    #340 #376 #381 Translations update from Kuma Weblate (Thanks @dng-nguyn @401U @Alanimdeo @MrEddX @NexiaMoe @Yirade @Zaid-maker @catfishlty @cyril59310 @gergepalfi @marco-doerig @nazo6 @oplexz @pacoculebras @queeup @stanol @tomeczekstecc)
+    #344 Added Hungarian (Thanks @gergepalfi)
+    #377 Added catalan (Thanks @pacoculebras)
+    #368 Added translate key (Thanks @cyril59310)
+
+  Please let me know if your username is missing, if your pull request has been merged in this version, or your commit has been included in one of the pull requests.
+
+developer: Louis Lam (@louislam)
+submitter: FlyinPancake
+submission: https://github.com/getumbrel/umbrel-apps/pull/10000
```

```suggestion
submission: https://github.com/getumbrel/umbrel-apps/pull/1106
```

You forgot to update the PR number